### PR TITLE
[WIP] Allow Colors customization in embeddable Object Browser

### PR DESCRIPTION
**Pull request by bexsoft**

### CODE REVIEW

File: portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx [hunk 1]
```diff
@@ -109,6 +110,8 @@ const ListBuckets = ({ classes }: IListBucketsProps) => {
   const features = useSelector(selFeatures);
   const obOnly = !!features?.includes("object-browser-only");
 
+  console.log(getCookieValue("eb_st"));
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/ListBuckets.tsx:113]:
```suggestion
```

**Comment by harshavardhana** [comment]
how does this customization work?

**Comment by bexsoft** [comment]
> how does this customization work?

Still WIP, this will only be available for embbeded object browser, you need to add one more query param with custom values for this.

**Comment by bexsoft** [comment]
Closing as implementation will change. Will make a new PR for this functionality

**Pull request closed without merging**